src/plugins/theme/colors.js

```diff
@@ -4,16 +4,14 @@
  */
 module.exports = function () {
   return {
-    transparent: 'transparent',
-    current: 'currentColor',
-    inherit: 'inherit',
     // Primary Colors
     // https://identity.stanford.edu/design-elements/color/primary-colors/
     'cardinal-red': {
       DEFAULT: '#8C1515', // Stanford main brand color
       light: '#B83A4B',
       dark: '#820000',
     },
+    // These replace the Tailwind default Black spectrum
```

Comment:
3 places where our Decanter color names overlaps with TW color names - black, sky and stone. I've added comments in the code. I think we should override the TW ones with ours since it is our official palette from the Identity Guide.